common/views/components/styled/AlignFont.tsx

```diff
@@ -0,0 +1,30 @@
+import { useContext, FunctionComponent, ReactNode } from 'react';
+import styled from 'styled-components';
+import TogglesContext from '@weco/common/views/components/TogglesContext/TogglesContext';
+
+type Props = {
+  className?: string;
+  children: ReactNode;
+};
+
+const Align = styled.span<{ isOn: boolean }>`
+  .fonts-loaded & {
+    ${props =>
+      props.isOn &&
+      `
+      transform: translateY(-${props.theme.fontVerticalOffset});
+      display: inline-flex;
+    `}
+  }
+`;
+
+const AlignFont: FunctionComponent<Props> = ({ children, className }) => {
+  const { helveticaRegular } = useContext(TogglesContext);
+  return (
+    <Align isOn={helveticaRegular} className={className}>
+      {children}
+    </Align>
+  );
+};
+
+export default AlignFont;
```

Comment:
Could we add a comment about when we should use this and why it's needed?

Reply:
yeah, good point